client/public/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
What is this for?

Reply:
don't know, probably useless but it was there before